docker/plugins.txt

```diff
@@ -1,81 +1,86 @@
 ace-editor:1.1
 amazon-ecs:1.37
 antisamy-markup-formatter:2.1
-apache-httpcomponents-client-4-api:4.5.10-2.0
+apache-httpcomponents-client-4-api:4.5.13-1.0
 authentication-tokens:1.4
 aws-credentials:1.28
-aws-java-sdk:1.11.854
+aws-java-sdk:1.11.955
+bootstrap4-api:4.6.0-1
 bouncycastle-api:2.18
 branch-api:2.6.3
 build-timeout:1.20
+checks-api:1.5.0
 cloudbees-folder:6.15
 command-launcher:1.5
-configuration-as-code:1.46
+configuration-as-code:1.47
 configuration-as-code-secret-ssm:1.0.1
-credentials:2.3.14
+credentials:2.3.15
 credentials-binding:1.24
-cucumber-reports:5.4.0
+cucumber-reports:5.5.0
 display-url-api:2.3.4
 docker-commons:1.17
 docker-workflow:1.25
 durable-task:1.35
-echarts-api:4.9.0-2
-git:4.4.5
-git-client:3.5.1
+echarts-api:4.9.0-3
+font-awesome-api:5.15.2-1
+git:4.6.0
+git-client:3.6.0
 git-server:1.9
-github:1.32.0
-github-api:1.116
-github-branch-source:2.9.1
+github:1.33.1
+github-api:1.122
+github-branch-source:2.10.1
+github-oauth:0.34-SNAPSHOT (private-53e10daa-sam)
```

Comment:
@MancunianSam When I exported the plugins, it included your fork of github-oauth. Should it be included in the list or not? What happens when it's not in the list and you redeploy the Jenkins ECS task?

Reply:
Yeah I think it is needed otherwise it doesn't bother installing it from the built jar.

Reply:
Any idea why it wasn't in the list before?

Reply:
Not sure, best guess is that it was installed manually each time but that seems unlikely. I could be wrong about it being needed, you could try and take it out and see if it still installs it based on the jar location in the docker image.

Reply:
Looks like we do need to remove it from the list, because the Docker build can't install this private fork version.

I've removed it, and also updated the Readme so to include the `--pull --no-cache` flags.